docs/user-manuals/cloneset.md

```diff
@@ -606,6 +612,9 @@ spec:
 
 ![Lifecycle circulation](/img/docs/user-manuals/cloneset-lifecycle.png)
 
+- When CloneSet create a Pod (including scale in and recreate update):
+  - The Pod will be regarded as `Available` and util the `PreNormal` hook is satified.
+  - This is very useful for some post-checks during Pod creation, for example, you can check if the Pod has been mounted to the SLB backend to avoid traffic loss caused by new instance mounting failure after the old instance is destroyed during rolling upgrades.
```

Comment:
> This is very useful for some post-checks during Pod creation, for example, you can check if the Pod has been mounted to the SLB backend to avoid traffic loss caused by new instance mounting failure after the old instance is destroyed during rolling upgrades.

preNormal hook can be used for post-checks after pod creation. For example, one can check if the pod have been added as the SLB backends successfully. Without preNormal hook,  one may encounter traffic loss during rolling upgrades  if CCM fails to add new pods to the SLB backends.